Makefile

```diff
@@ -40,12 +40,6 @@ REGISTRY ?= quay.io/open-cluster-management
 TAG ?= latest
 IMAGE_NAME_AND_VERSION ?= $(REGISTRY)/$(IMG)
 
-# Fix sed issues on mac by using GSED
-SED="sed"
-ifeq ($(GOOS), darwin)
-  SED="gsed"
-endif
-
```

Comment:
I'm surprised this would need to be removed... it might've been a while ago, but do you remember why this change was made?

Reply:
I don't think Jeffrey and I made this change (unless it was by accident). I'll look into this and get back to you on that.

Reply:
This is important. Plz don't remove. For mac this is important